y-ret-or-n-p.el

```diff
@@ -23,7 +23,12 @@
 ;; Public License.
 
 ;;; Commentary:
-;; See readme (http://github.com/sancoder-q/y-ret-or-n-p.el) for installation and usage
+;;
+;; | Command       | You have to type                        |
+;; |---------------+-----------------------------------------|
+;; | yes-or-no-p   | 'yes' or 'no', followed by RET          |
+;; | y-or-no-p     | 'y' or 'n', followed by RET             |
+;; | y-ret-or-no-p | RET to confirm straight away or 'n' RET |
```

Comment:
The package does not even define a function by that name. And I very much hope the function it does define does not behave the way you describe it. Making it *harder* to cancel than to confirm would be strange, after all canceling by error is usually less dangerous than confirming by mistake.